sdv/single_table/copulas.py

```diff
@@ -108,3 +108,18 @@ def _fit(self, processed_data):
         with warnings.catch_warnings():
             warnings.filterwarnings('ignore', module='scipy')
             self._model.fit(processed_data)
+
+    def _warn(self, column_name_to_transformer):
+        """Raise warnings of a model.
+
+        Args:
+            column_name_to_transformer (dict):
+                Dict mapping column names to transformers to be used for that column.
+        """
+        for column in column_name_to_transformer:
+            sdtype = self.metadata._columns[column]['sdtype']
+            if sdtype == 'categorical':
+                warnings.warn(
```

Comment:
I think there should be one more warning like this on the other models for when categorical  or boolean transformers are set